.github/workflows/publish-testing.yml

```diff
@@ -0,0 +1,137 @@
+name: Publish to Testing
```

Comment:
question: would we have some similar action for `release` stage or something else?

Reply:
No, the `stable` release process will still be the responsibility of Voulage, and it will be an one off operations. For example, when time comes to release Regolith v3.3 (GA, beta, etc), we just need to copy the content of `testing/` folder into corresponding `release-X_Y/` folder and run the deb-builder workflow.